.build-tools/pkg/metadataschema/validators.go

```diff
@@ -135,6 +143,52 @@ func (c *ComponentMetadata) AppendBuiltin() error {
 				},
 			},
 		)
+	case mdutils.BindingType:
+		if c.Binding != nil {
+			if c.Metadata == nil {
+				c.Metadata = []Metadata{}
+			}
+
+			var direction string
+
+			switch {
+			case c.Binding.Input && c.Binding.Output:
+				direction = BINDING_DIRECTION_BOTH
+			case c.Binding.Input:
+				direction = BINDING_DIRECTION_INPUT
+			case c.Binding.Output:
+				direction = BINDING_DIRECTION_OUTPUT
+			}
+
+			c.Metadata = append(c.Metadata,
+				Metadata{
+					Name:        BINDING_DIRECTION_METADATA_KEY,
+					Type:        "string",
+					Description: "Indicates the bindings direction of the component.",
+					Example:     direction,
+					URL: &URL{
+						Title: "Documentation",
+						URL:   "https://docs.dapr.io/reference/api/bindings_api/#binding-direction-optional",
```

Comment:
This link isn't working for me - it doesn't point to the section specified here

Reply:
I will merge this PR once the docs PR is merged, so docs maintainer confirm the URL